app/src/test/java/com/brainwallet/tools/util/LocaleHelperTest.kt

```diff
@@ -56,27 +56,29 @@ class LocaleHelperTest {
     fun `check all language codes`() {
         Language.entries
             .also { languages ->
-                assertEquals(15, languages.size)
+                assertEquals(17, languages.size)
             }
             .map { it.code }
             .also { langCodes ->
                 assertEquals(
                     listOf(
                         "en",
-                        "de",
                         "es",
-                        "fr",
                         "in",
-                        "it",
-                        "pt",
-                        "tr",
+                        "ar",
                         "uk",
                         "ru",
+                        "pt",
                         "ko",
+                        "fr",
+                        "zh-TW",
+                        "tr",
                         "ja",
+                        "de",
                         "zh-CN",
-                        "zh-TW",
-                        "ar"
+                        "hi",
+                        "it",
+                        " " ///Hack: implementation and allow the user to pick the last language
```

Comment:
could you pls explain about this? @kcw-grunt

Reply:
Oh the language Recycler view was blocking the last list item.  But we are not using it anymore @andhikayuana